templates/docs/examples/layouts/empty-state/no-content.html

```diff
@@ -0,0 +1,33 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Empty State / No content {% endblock %}
+
+{% block standalone_css %}layouts{% endblock %}
+
+{% block content %}
+<table aria-label="Vanilla framework no content empty state">
+    <thead>
+        <tr>
+            <th>Vendor ID</th>
+            <th>Product</th>
+            <th>ProductID</th>
+            <th>Driver</th>
+            <th>Huma Node</th>
+            <th>PCI Address</th>
+        </tr>
+    </thead>
+    <caption>
```

Comment:
@bethcollins92 Do you know if that's fine from accessibility point of view or for screen readers? Seems a lot to put into table 'caption', but maybe it's fine. It is a valid HTML and seems OK from semantics point of view I guess.

Reply:
I checked it with voice over and it seems to only read the caption and not the column headers, which is what we want/makes sense as there is no point reading them if there is no content. I can't see anything that would make this incorrect semantics wise either